docs/Manual Build Instructions.md

```diff
@@ -0,0 +1,92 @@
+## Note to the End User
+Manual builds are not recommended for end users of the Scan Engine and EtherCAT Custom Device. The build process is time consuming and requires three repositories to be cloned and built.
+
+Most end users should find the latest builds of the Scan Engine and EtherCAT Custom Device for supported versions of VeriStand at [Scan Engine and EtherCAT Custom Device Releases page](https://github.com/ni/niveristand-scan-engine-ethercat-custom-device/releases). These are automatically built, tested, and packaged into NI Packages.
+
+## Manual Build Instructions for the Scan Engine and EtherCAT Custom Device for Developers and Contributors
+Follow these steps to manually build all of the components necessary to load and use the Scan Engine and EtherCAT Custom Device.
+
+### Version Management
+All projects must be open and build in the version of LabVIEW that corresponds to the VeriStand version being targeted.
+
+For example, building for VeriStand 2020 requires using LabVIEW 2020 to build each build specification.
+
+Glossary of Terms:
+- PPL = Packed Project Library designated by the file extension .lvlibp
+- LLB = LabVIEW Library designated by the file extension .llb
```

Comment:
You might want to clarify that LLBs are build artifacts. It's not clear what the difference is from a `.lvlib` from this description.